src/main/resources/assets/styles/_upcomingReleases.scss

```diff
@@ -99,23 +99,30 @@
   }
 
   @include media-breakpoint-up(md) {
-    time {
-      display: block;
+    .time-wrapper{
+      display: table;
       border: 2px solid $ssb-dark-5;
       width: 97px;
-      height: 80px;
-      margin: 8px 20px 3px 0;
-      padding: 7px;
+      max-height: 80px;
+      margin: .5rem 1.25rem 0.1rem 0;
       text-align: center;
-      background: white;
-      color: $ssb-dark-5;
-
-      .day {
-        font-size: 40px;
-        font-family: $font-roboto-condensed-bold;
-        line-height: 1;
-        display: block;
+
+      time {
+        display: table-cell;
+        vertical-align: middle;
+        margin: .5rem;
+        background: white;
```

Comment:
Very nitpicky, and you can skip it if you'd like, but consider using the designsystem color "white" (#fff).  
https://design.ssb.no/#/components/color